chevah_build

```diff
@@ -153,48 +190,42 @@ case $OS in
             else
                 export LDFLAGS="$LDFLAGS -L/usr/lib/mps/64 -R/usr/lib/mps/64"
             fi
+        else
+            # On Solaris 11 we can build the new libraries.
+            EXTRA_LIBRARIES=$EXTRA_LIBRARIES_CFFI
+            PIP_LIBRARIES=$PIP_LIBRARIES_EVERYWHERE
         fi
+        BUILD_CFFI="yes"
```

Comment:
hm.... in master CFFI is build for any solaris version ... so I think that this is just keeping the behavior from master